src/connectors/soundclick.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const filter = new MetadataFilter({
+	track: cleanupTrack
+});
+
+Connector.playerSelector = '#playerWrap';
+
+Connector.artistSelector = '#playerSCLK_artistName';
+
+// use playlist to get track title
+// to avoid missing content when quotation marks present
+Connector.getTrack = () => {
+	const text = Util.getTextFromSelectors('.songActive .playlist-songtitle');
+	const artist = Connector.getArtist();
+	return text.substr(text.match(artist).index + artist.length + 3);
```

Comment:
Can we use `.songActive .playlist-songtitle` as an `artistTrackSelector` instead? What do you think?

Alternatively, you could put removing an artist name to `cleanupTrack` function.